.github/workflows/npm-release.yml

```
@@ -0,0 +1,58 @@
+          - sps-frontend # todo: muss angepasst werden. eigenen Ordner verwenden
+
+permissions:
+
+  contents: write
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # to be able to publish a GitHub release
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)